test/inspector/inspector-helper.js

```diff
@@ -215,10 +215,10 @@ TestSession.prototype.sendInspectorCommands = function(commands) {
       timeoutId = setTimeout(() => {
         let s = '';
         for (const id in this.messages_) {
-          s += id + ', ';
+          s += `${id}, `;
```

Comment:
+1 to @refack's suggestion again. While you're here you might as well simplify, and it's a simple atomic change (so not too painful to review).